en/includes/references/conditional-auth/api-reference.md

```diff
@@ -1089,15 +1060,76 @@ It contains the necessary metadata for invoking the API when calling the callCho
   </tr>
 </table>
 
-If the consumer key and the consumer secret are added as secrets, they should be included in the ConnectionMetadata as aliases, as shown below.
+You can securely store consumer key and consumer secrets as secrets in conditional authentication script in the Asgardeo Console and refer to
+them in your conditional authentication scripts using the `secrets.key` syntax. For example, to retrieve a secret value, you can use:
+```angular2html
+var consumerSecret = secrets.clientSecret;
+```
+For more information on adding secrets, refer to the [Add a secret to the script]({{base_path}}/guides/authentication/conditional-auth/configure-conditional-auth/#add-a-secret-to-the-script) section in the 
+documentation.
+
+??? note "Before 30th of June 2024, secrets can be consumed as below and it's possible to access secrets in the same way"
```

Comment:
before 30th we can only access it like below.
But after 30th we don't recommend to use this way, rather use secrets.key way.
But for the people who use this can still use it even after 30th

Reply:
Let's reword the note to the following

"Change in behavior from 30th June 2024"

Starting from 30th June 2024, you are no longer required to set the aliases for consumer keys and consumer secrets when calling the callChoreo command. Instead, you may directly reference them using the `secrets.secretName` notation. 

However, if you prefer, you may continue using the previous method.

Reply:
Addressed with https://github.com/wso2/docs-is/pull/4660/commits/786c26614664c98ec1c21d1f25a65f0715b85973